### ISSUE TITLE
Huge conflicts?

### ISSUE THREAD

**Issue by XanthosTheoric** [chzesa/tiled-tab-groups#14]
So, I've only recently discovered your add-on and I believe first you deserve one hell of a good pat on the back.  I love the system!

I only wish of course, that this was cross platform so that I could use your add-on in other browsers too.  Although, I'm sorry if this is available and I just haven't noticed.

Anyway!  Problem.  So I'm finding that while running "Tree Style Tab" and "Tiled Tab Groups", both of which yours, I find almost any other add-on loaded will result in numerous problems.

Handy List:
1. Loading other add-ons into Firefox while using these is a hit-or-miss if this results in the inability to load any website within Firefox.  It just hangs endlessly in a loading state.  Unsure why, and this happens to add-ons which seemingly have nothing to even do with yours like Grammarly or AdBlock Pro.
2. Within either of the settings menu for both add-ons I'm noticing that my settings per add-on weren't being saved on close.  After reviewing my settings for Firefox to ensure that cashe wasn't being dumped on close or anything, it seems that the add-ons just aren't holding onto the options.  Without an on-screen, ( or one I've missed ), save button I can't even be sure that I'm applying the changes so this one might be on me.
3. On the exterior while both are enabled I noticed the tiled add-ons icon doesn't even show.  When clicked it displays a tiny empty tool-tip, and when forced open it features none of my save groups or the pages within.  At this juncture I'm unsure where they are, if they're hidden, unloaded, or whatever.  Not sure how I'd find out.
4. When on the browsers normal display area, the settings for the tree-style tab such as being on the left are also dumped every time I close Firefox.

Well.  Thinking about it now, a list wasn't really needed.  I seem to have only four issues I can think of right now.

**Comment by chzesa** [comment]
Thank you for the kind words. Firefox is the only browser with support for tab hiding so it isn't possible to develop the addon for other browsers.

[Tree style tab](https://addons.mozilla.org/firefox/addon/tree-style-tab/) isn't developed by me, I develop a similar addon called [ftt](https://addons.mozilla.org/firefox/addon/ftt/). Tiled tab groups has some compatibility features I implemented back when I was using tree style tabs

I'm going to need some more information:
* Which version of firefox are you using? Are you using private browsing windows?
* Do the addon loading problems occur when only one of the two addons is enabled?
* Do the problems occur when using a new profile? [Instructions for managing profiles can be found here](https://support.mozilla.org/kb/profile-manager-create-remove-switch-firefox-profiles)
* Have you enabled any of the settings under the Tree style tab section of tiled tab groups?
* Has the addon emitted any error messages? This can be checked by entering `about:debugging#/runtime/this-firefox` and inspecting the respective addon

**Comment by XanthosTheoric** [comment]
Ok.  I can't seem to find a way to contact you privately, so out to open air.

I have discovered with the Firefox team what was going on exactly.  It's a bit of a known bug and they talked me through how to fix it.

So, your addon's as you already know are both in full working order.  Sort of need this closed as I've discovered that there is absolutely no error with your addon.  I don't think I ever fully moved through our interactions and indirectly I ended losing contact?  I'm not sure.  I was sure that this was still open though, wanted to get this nugget on here.

Now that my Firefox is fixed, I've been having a blast with your add-on.  You made something special and it's made it so much easier to deal with my OCD, and keeping projects separate without having to bookmark and close tabs.  Using the grouping system, nesting, and the like revolutionizes how I've come to use my browser.

I dare say that I believe your addon is as revolutionary of an update to the browsing experience that it's comparable to the level of change that being able to use tabs instead of an army of windows.  Be ready to get copied on other browsers when they are compatible with such a concept.  Get the copyright now before it's too late, lol.

All the best, and I really appreciated you getting in touch with me extremely quickly about this issue, and again I'm sorry that I kind of left you adrift like that.  Be well, and once again, I wish you all the best in the new year that lays before us.  Keep up the good work, you're one of those people who have good ideas and the drive to make them happen.  It's a good mind-set and it'll serve you well.

Cheers!
